packages/core/src/routes/interaction/verifications/mfa-payload-verification.ts

```diff
@@ -232,7 +232,7 @@ export async function verifyMfaPayloadVerification(
       // Update the authenticator's counter in the DB to the newest count in the authentication
       await tenant.queries.users.updateUserById(accountId, {
         mfaVerifications: user.mfaVerifications.map((mfa) => {
-          if (mfa.type !== MfaFactor.WebAuthn) {
+          if (mfa.type !== MfaFactor.WebAuthn || mfa.id !== result.id) {
```

Comment:
Is this related to the `lastUseTime` updates? Don't have much context for this update.

Reply:
This is the WebAuthn's "counter" updates for some security device.

Reply:
Year, it's a bug fix that should not be present in this PR, but since is tiny...